src/app/dungeon/entities/wall.component.ts

```diff
@@ -1,41 +1,37 @@
 import { ChangeDetectionStrategy, Component, computed, CUSTOM_ELEMENTS_SCHEMA, effect, input } from '@angular/core';
 import { extend } from 'angular-three';
-import { NgtrCuboidCollider } from 'angular-three-rapier';
-import { injectTexture } from 'angular-three-soba/loaders';
+import { NgtrCuboidCollider, NgtrRigidBody } from 'angular-three-rapier';
+import { textureResource } from 'angular-three-soba/loaders';
 import { BoxGeometry, Mesh, MeshBasicMaterial, NearestFilter, Object3D, RepeatWrapping, Vector3 } from 'three';
 
 @Component({
   selector: 'dungeon-wall',
   template: `
-    <ngt-mesh ngtrRigidBody="fixed" [position]="positionVector()">
-      <ngt-box-geometry />
-      <ngt-mesh-basic-material [map]="wallMap()" />
-      <ngt-object3D ngtrCuboidCollider [args]="[0.5, 0.5, 0.5]" />
-    </ngt-mesh>
+    <ngt-object3D rigidBody="fixed" [position]="positionVector()">
+      <ngt-mesh>
+        <ngt-box-geometry />
+        <ngt-mesh-basic-material [map]="wallMap()" />
+      </ngt-mesh>
+
+      <ngt-object3D [cuboidCollider]="[0.5, 0.5, 0.5]" />
+    </ngt-object3D>
```

Comment:
rigidBody can only be on `ngt-object3D`; wrap the object 3D with a mesh if you want actual presentation.